contracts/Escrow.sol

```diff
@@ -190,10 +190,8 @@ contract Escrow is IERC777Recipient, IERC777Sender, Permissions {
         onlyBeneficiary
     {
         Allocator allocator = Allocator(contractManager.getContract("Allocator"));
-        require(allocator.isVestingActive(_beneficiary), "Beneficiary is not Active");        
-        if (!allocator.isDelegationAllowed(_beneficiary)) {
-            require(allocator.calculateVestedAmount(_beneficiary) >= amount, "Incorrect amount to delegate");
-        }
+        require(allocator.isDelegationAllowed(_beneficiary), "Delegation is not allowed");
```

Comment:
So beneficiary could not delegate through Escrow if tokens are vested and plan is not delegatable?

Reply:
It still can delegate after retrieving funds.
I think more strict constraint is better because of 2 reasons:
1. It makes implementation simpler
2. delegating of vested tokens in fact put the foundation on risk because is case of slashing beneficiary may not be slashed.